scripts/install_third_party.sh

```diff
@@ -101,3 +101,26 @@ if [ ! -d "$TOOLS_DIR/numpy-1.6.1" ]; then
 
   pip install numpy==1.6.1 --target="$TOOLS_DIR/numpy-1.6.1"
 fi
+
+echo Checking if pylint is installed in $TOOLS_DIR/pip_packages
+if [ ! -d "$TOOLS_DIR/pylint-1.5.2" ]; then
+  echo Installing Pylint
+
+  if ! type pip > /dev/null 2>&1 ; then
```

Comment:
Let's not repeat all this. Could you instead do:

- check if pip exists, and if not, print an error message
[from now on, we assume pip does exist]
- check if numpy exists, and if not, install it using pip
- check if pylint exists, and if not, install it using pip